model/gen-ai/events.yaml

```
@@ -30,6 +70,79 @@ groups:
     brief: >
       This event describes the assistant message passed to GenAI system or received from it.
```

```suggestion
      This event describes the assistant instructions passed inside the prompt to the GenAI model.
```
On this one, I thought the expectation is not that the response from the model is added as a `gen_ai.assistant.message` event. Rather, this is a previous `gen_ai.choice` added into a new prompt to create a multi-turn interaction in the stateless communication.  The "or received from it" at the end of the sentence could be confusing.

---

fixed in https://github.com/open-telemetry/semantic-conventions/pull/1469/commits/a25e181f025d7f195bddb3691930f757e8c1f4bf with the same caveat about "passed inside the prompt"

instructions can be misleading -it's a synonym to system message